src/xcode/ENA/ENA/Source/Scenes/CreateAntigenTestProfile/Cells/CreateAntigenTestProfileInputCell.swift

```diff
@@ -35,6 +35,7 @@ class CreateAntigenTestProfileInputCell: UITableViewCell, ReuseIdentifierProvidi
 
 		birthdayPicker = UIDatePicker()
 		birthdayPicker.locale = Locale(identifier: "de_DE")
+		birthdayPicker.timeZone = TimeZone.current
```

Comment:
just wonder why the picker locale is set to 'de_DE' - don't we support other settings as well?

Reply:
@KaiTeuber I was wondering that myself but I don't know.